atom/common/node_bindings.cc

```diff
@@ -194,6 +199,10 @@ node::Environment* NodeBindings::CreateEnvironment(
       base::CommandLine::ForCurrentProcess()->HasSwitch("debug-brk"))
     process.Set("_debugWaitConnect", true);
 
+  process.SetMethod(
+    "setProcessTitle",
```

Comment:
I didn't call this `setTitle` so that it wouldn't be construed as being equivalent to `process.title = 'whatever'`.